modules/core/src/main/java/org/apache/synapse/config/xml/EnrichMediatorFactory.java

```diff
@@ -119,8 +132,7 @@ private void populateSource(Source source, OMElement sourceEle) {
 
             if (inlineElem != null) {
                 source.setInlineOMNode(inlineElem);
-            } else if (sourceEle.getText() != null
-                    && (!sourceEle.getText().equals(""))) {
+            } else if (sourceEle.getText() != null && (!sourceEle.getText().equals(""))) {
```

Comment:
we don't need two comparisons if we do like this:
if (!("").equals(sourceEle.getText()))

Reply:
https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isBlank-java.lang.CharSequence-

- This also has similar effect ( but may be we don't use StringUtils in this code base)

Reply:
`(!("").equals(sourceEle.getText()))` this only check whether sourceEle.getText() is empty or not, but `sourceEle.getText() != null` this will check whether sourceEle.getText() is null or not, so we need two comparisons to check both.

Reply:
@Nirthika  yes you are correct. thanks for the explanation.
StringUtils option is also can use here **only if we already using** commons-lang within EI. otherwise IMHO what you have done here is correct

Reply:
@ramith yes we have **commons-lang** dependency but we don't use **StringUtils** in this code base. We can use `!StringUtils.isBlank(sourceEle.getText())` instead of 
`sourceEle.getText() != null && (!sourceEle.getText().equals(""))`